MovieMVP/SceneDelegate.swift

```diff
@@ -3,19 +3,24 @@
 
 import UIKit
 
-// MARK: - SceneDelegate
-
+/// Сцена
 final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+    // MARK: - Public Properties
+
     var window: UIWindow?
 
+    // MARK: - Public Methods
+
     func scene(
         _ scene: UIScene,
         willConnectTo session: UISceneSession,
         options connectionOptions: UIScene.ConnectionOptions
     ) {
-        let listMoviesViewController = ListMoviesViewController()
-        let mainNavigationController = UINavigationController(rootViewController: listMoviesViewController)
-        window?.rootViewController = mainNavigationController
+        let navigationController = UINavigationController()
+        let assembleBuilder = AssembleBulder()
+        let router = Router(navigationController: navigationController, assembleBulder: assembleBuilder)
+        router.initialViewController()
+        window?.rootViewController = navigationController
         window?.backgroundColor = UIColor.white
```

Comment:
```suggestion
        window?.backgroundColor = UIColor.white
```
Лучше в контролере цвет укажи